src/backend/parser/Make.js

```diff
@@ -0,0 +1,13 @@
+var JisonLex = require('jison-lex');
+var Jison = require('jison');
+var fs = require('fs');
+
+var lexerGrammar = fs.readFileSync('src/backend/parser/ansic.jisonlex', 'utf-8');
+var lexerSource = JisonLex.generate(lexerGrammar);
+fs.writeFileSync('src/backend/parser/ansic.lex.js', lexerSource);
+
+var parserGrammar = fs.readFileSync('src/backend/parser/ansic.jison', 'utf-8');
```

Comment:
"Compiled" files are to be put in 'distrib' folder - this should be configured from Gruntfile